torch/nn/modules/loss.py

```diff
@@ -172,13 +175,15 @@ class NLLLoss2d(NLLLoss):
 
     Examples::
 
-        >>> m = nn.Conv2d(16, 32, (3, 3)).float()
+        >>> N, C = 5, 4
         >>> loss = nn.NLLLoss2d()
         >>> # input is of size N x C x height x width
-        >>> input = autograd.Variable(torch.randn(3, 16, 10, 10))
+        >>> data = Variable(torch.randn(N, 16, 10, 10))
+        >>> m = nn.Conv2d(16, C, (3, 3)).float()
+        >>> input = m(data)
```

Comment:
It's not really an input. I think it's better to keep it as `loss(m(input), target)`

Reply:
done